src/sherwood.cr

```diff
@@ -95,6 +95,15 @@ class Sherwood
 
       # TODO: Variable Operations
       # TODO: Control Flow
+      when 0x70 then insp -= popType(SWInt, stack)
+      when 0x71 then
+        if popType(Bool, stack) then {
+          popType(SWInt, stack)
+        } else {
```

Comment:
This empty `else` is completely unnecessary.

Reply:
Oh. 
I don't know how Crystal works, but could figure most stuff out. I looked at a tutorial which said it was required. I did not to take any chances 😄